pkgs/by-name/ch/choose-gui/package.nix

```
@@ -0,0 +1,48 @@
+{ lib, stdenv, system, fetchFromGitHub, xcbuild, coreutils, darwin }:
```

```suggestion
{ lib, stdenv, fetchFromGitHub, xcbuild, coreutils, darwin }:
```
`system` is still there, you need to remove it.

---

Sorry, I forgot to push from my local machine -- latest commit fixes this :smile: